specification/ai/Azure.AI.Projects/evaluations/models.tsp

```diff
@@ -176,3 +179,75 @@ model AgentEvaluation {
   @doc("The agent evaluation result.")
   result?: Array<AgentEvaluationResult>;
 }
+
+@doc("Abstract base model representing a single message in a conversation.")
+@discriminator("role")
+@added(Versions.v2025_05_15_preview)
+model Message {
```

Comment:
It seems to be not evaluation specific, can we move it to somewhere relates to chat completion?